assets/javascripts/discourse/components/ai-persona-editor.gjs

```diff
@@ -456,6 +438,19 @@ export default class PersonaEditor extends Component {
           />
         </div>
       {{/if}}
+      <div class="control-group">
+        <label>{{I18n.t "discourse_ai.ai_persona.max_context_posts"}}</label>
+        <Input
+          @type="number"
+          lang="en"
+          class="ai-persona-editor__max_context_posts"
+          @value={{this.editingModel.max_context_posts}}
+        />
+        <DTooltip
+          @icon="question-circle"
+          @content={{I18n.t "discourse_ai.ai_persona.max_context_posts_help"}}
+        />
+      </div>
       <div class="control-group">
```

Comment:
I noticed in the UI Temperature and Top P are kind of grouped together; it's happening because they are in the same `control-group` div, just need to split it into 2:

![image](https://github.com/discourse/discourse-ai/assets/920448/0f7ba68e-158d-4d9d-85ca-35a99ac413b8)

Reply:
cool will sort this.